app/controllers/hooks_controller.rb

```diff
@@ -15,6 +15,7 @@ def github
   end
 
   def package
+    Rails.logger.info "HooksController#package platform=#{params['platform']} name=#{params['name']} param_keys=#{params.keys.join(',')}"
```

Comment:
Wanted to see how frequent this project hook is getting called for this failing PlatformIO project.